src/test/java/com/pubnub/contract/objectV2/step/Utils.kt

```diff
@@ -0,0 +1,13 @@
+package com.pubnub.contract.objectV2.step
+
+import com.google.gson.Gson
+import com.pubnub.api.models.consumer.objects_api.uuid.PNUUIDMetadata
+import com.pubnub.contract.CONTRACT_TEST_CONFIG
+import java.io.File
+
+fun loadPersona(personaName: String): PNUUIDMetadata {
```

Comment:
I renamed it to `loadPersonaUUIDMetadata` to avoid problems with name clash between `User` and `UUID` personas type

Reply:
Ok, I can rename it to be the same.